.clangd

```diff
@@ -0,0 +1,4 @@
+Style:
+  AngledHeaders:
+    # public headers (under include/seastar) use angle brackets
+    - "seastar/.*"
```

Comment:
How will this handle `#include "zlib.h"` or similar?

Reply:
This rule is effectively just an override for the seastar case where the headers are _not_ included via a system include path, yet use <>.